website/static/js/config.js

```diff
@@ -0,0 +1,153 @@
+  elementID: 'klaro',
+
+  // How Klaro should store the user's preferences. It can be either 'cookies'
+  // or 'localStorage'. If undefined, Klaro will use cookies.
+  klaroStorage: 'localStorage',
+
+  // You can customize the name of the cookie that Klaro uses for storing
+  // user consent decisions. If undefined, Klaro will use 'klaro'.
+  cookieName: 'substrateDevHubCookie',
+
+  // You can also set a custom expiration time for the Klaro cookie.
+  // By default, it will expire after 120 days.
+  cookieExpiresAfterDays: 120,
+
+  // You can customize the name of the cookie that Klaro will use to
+  // store user consent. If undefined, Klaro will use 'klaro'.
+
+  // Put a link to your privacy policy here (relative or absolute).
+  privacyPolicy: 'https://www.parity.io/privacy/',
+
+  // Defines the default state for applications (true=enabled by default).
+  default: true,
+
+  // If "mustConsent" is set to true, Klaro will directly display the consent
+  // manager modal and not allow the user to close it before having actively
+  // consented or declines the use of third-party apps.
+  mustConsent: false,
+
+  // You can define the UI language directly here. If undefined, Klaro will
+  // use the value given in the global "lang" variable. If that does
+  // not exist, it will use the value given in the "lang" attribute of your
+  // HTML tag. If that also doesn't exist, it will use 'en'.
+  //lang: 'en',
+
+  // You can overwrite existing translations and add translations for your
+  // app descriptions and purposes. See `src/translations.yml` for a full
+  // list of translations that can be overwritten:
+  // https://github.com/DPKit/klaro/blob/master/src/translations.yml
+
+  // Example config that shows how to overwrite translations:
+  // https://github.com/DPKit/klaro/blob/master/src/configs/i18n.js
+  translations: {
+    // If you erase the "consentModal" translations, Klaro will use the
+    // defaults as defined in translations.yml
+    de: {
+      consentModal: {
+        description:
+          'Hier können Sie einsehen und anpassen, welche Information wir über Sie sammeln. Einträge die als "Beispiel" gekennzeichnet sind dienen lediglich zu Demonstrationszwecken und werden nicht wirklich verwendet.',
+      },
+      googleAnalytics: {
+        description: 'Sammeln von Besucherstatistiken',
+      },
+      cloudflare: {
+        description: 'Schutz gegen DDoS-Angriffe',
+      },
+      purposes: {
+        analytics: 'Besucher-Statistiken',
+        security: 'Sicherheit',
+      },
+    },
+    en: {
+      consentModal: {
+        description:
+          'We use necessary cookies to make our site work. We\'d also like to set optional tracking mechanisms to help us improve it by collecting and reporting information on how you use it. Here you can customise the information we collect about you by giving or revoking your consent to our use of optional tracking mechanisms.',
```

Comment:
What "necessary cookies" do we use? Isn't this just about google analytics which we're allowing the user to opt out of?

Reply:
Yeah, without cloudflare, I think there are no "necessary" cookies.

This means we should probably remove the first sentence, and change the two buttons on the original klaro prompt to represent you either allow GA or you allow nothing.

Reply:
The current text in buttons is consistent with parity.io and kusama.network (it also has GA only).
The button can be renamed by text changes in the klaro.min.js. Kirill @kirushik may be the person can help the decision on it.

Reply:
Yeah, let's have Kirill decide what to put

Reply:
I've seen we're thinking about integrating substrate playground into substrate.dev.
Since it will feature logins, it would have to rely on some cookies mechanism, right?
So we either keep this in place for now, or we would have to rebuild klaro while integrating playground in our site.
Keeping it seems to be an easier solution, both because it requires less effort, and because there's a lower chance to forget it and go to production with inaccurate disclaimers.